### DISCUSSION TITLE
QtCreator for Windows 8.1

### DISCUSSION THREAD

**Discussion by ghost** [rochus-keller/LeanCreator#11] [General]
Last QtCreator version could be built with Qt4 is QtCreator 3.2.2. Last QtCreator version could be built with Qt5 is QtCreator 7 (but they said the last QtCreator release that was built with Qt5 is QtCreator 5). The last QtCreator version allowed you to skip signing in Qt account is QtCreator 4.5.1. I remember I was trying to find a QtCreator version could run on my Windows 8.1 but not require me to sign in Qt account (I don't have any Qt account, I started with Qt on Linux and on Linux the QtCreator packaged by Linux distributions doesn't require you to sign in Qt account). I previously use the QtCreator package by the MSYS2 project but as a rolling release system they followed upstream and switched to Qt6 so I can no longer use their QtCreator. Then I found LeanCreator. It's a portable application and very lightweight, but as I need CMake support I then tried to download all Qt5 supported versions of QtCreator from their server (https://download.qt.io/archive/qtcreator/) and by trial and error finally I found QtCreator 4.5.1 is the last QtCreator version allowed me to skip signing in Qt account.

p/s: Later I found the location to download newer versions of QtCreator but I already found QtCreator 4.5.1 so I don't want to try anything further: https://download.qt.io/official_releases/qtcreator/

**Comment by rochus-keller** [comment]
If you need Qmake or Cmake LeanCreator is not for you. In future I might add a Cmake export to BUSY, but that's low priority.
